nbgrader/exchange/exchange.py

```diff
@@ -125,3 +125,35 @@ def start(self):
         self.init_src()
         self.init_dest()
         self.copy_files()
+
+    def get_current_user_courses(self):
+        """Check if student is enrolled in course"""
+        from jupyterhub.services.auth import HubOAuth
+        from tornado.httpclient import HTTPClient, HTTPRequest
+
+        if os.getenv('JUPYTERHUB_API_TOKEN'):
+            api_token = os.environ['JUPYTERHUB_API_TOKEN']
+        else:
+            self.exit("JUPYTERHUB_API_TOKEN env is required to run the exchange features of nbgrader.")
```

Comment:
@minrk can you please take a look if I'm using the APIs correctly?

Reply:
Use of the API looks absolutely right to me.